### ISSUE TITLE
Color Palette selection issues

### ISSUE THREAD

**Issue by Nekose** [Nekose/Mouseomate#5]
Running into issues selecting colors, likely due to different screen resolution affecting X,Y coordinates of color buttons. Will look into using pyautogui.locateOnScreen to grab each of the colors manually.

**Comment by Quadrum1** [comment]
Yes, that is an issue I knew was gonna come up, I just found no good way to locate the colors.
Maybe write a new subroutine to configure the settings and store them in a .json?

**Comment by dasheldequarius26** [comment]
Consider using `pyautogui.pixel` to sample color values at specific coordinates to verify if the button selections match the expected color outputs based on resolution.

**Comment by dameredezmond26** [comment]
Have you considered implementing a fallback mechanism in your color selection routine to handle cases where `pyautogui.locateOnScreen` fails to find the expected colors?

**Comment by claralima0462** [comment]
Instead of relying on screen coordinates, have you considered using OpenCV template matching within `pyautogui.locateOnScreen` to identify the buttons regardless of absolute position?
